src/docs/command-line.dox

```diff
@@ -142,7 +147,7 @@ which can be re-loaded into a new table using the \c load command.
 See @subpage dump_formats for details of the dump file formats.
 
 @subsection util_dump_synopsis Synopsis
-<code>wt [-RVv] [-C config] [-E secretkey ] [-h directory] dump [-jrx] [-c checkpoint] [-f output] uri</code>
+<code>wt [-RrVv] [-C config] [-E secretkey ] [-h directory] dump [-jrx] [-c checkpoint] [-f output] uri</code>
```

Comment:
Seeing all of these and the change above for `LRrVv` in the strings my question is: Should all of these synopsis lines have the `-L` added to them? Was that missed long ago?

Reply:
The command line documentation is definitely outdated as the whole thing is also missing -S. I'd prefer to not make refreshing the documentation part of this ticket and instead put it in in separate ticket.